src/lib/y2network/presenters/dns_summary.rb

```diff
@@ -54,6 +52,12 @@ def text
 
     private
 
+      def dhcp_hostname?
+        return false unless config.connections.any? { |c| c.bootproto&.dhcp? }
```

Comment:
Given that we are using `c.booproto&.dhcp?` more than once, what about adding a `dhcp?` method to the `ConnectionConfig` class?